### ISSUE TITLE
Unable to view documents

### ISSUE THREAD

**Issue by Elderglass** [bauerj/paperless_app#55]
**Describe the bug**
Unable to view existing documents
All requested permissions given. No storage permission requested?

**To Reproduce**
1. Open app
2. Click on any document
3. Stuck on 'Downloading document 0%'

**Expected behavior**
Document should load

**Screenshots**
![image](https://user-images.githubusercontent.com/79360160/135746375-73ac8cf6-ded4-4863-80cb-c079b5c06d2c.png)
![image](https://user-images.githubusercontent.com/79360160/135746370-6b9e56cb-fc2a-4bd3-aeaf-6dc6243369fa.png)

**Additional information**
*Paperless App Version*: e.g. 0.0.11
*Do you use Paperless NG as a backend*: Yes

**Comment by bauerj** [comment]
Hey,

that's odd. I assume it works fine in the browser? Do you use some kind of reverse proxy in front of the paperless server?

**Comment by Elderglass** [comment]
You're right it works absolutely fine in the browser on all devices. And no reverse proxy in use, just connecting directly to the ip of the paperless-ng docker instance.

**Comment by AdamGalt** [comment]
Was that log file useful? Happy to test and provide more logs if required. Just let me know.

**Comment by bauerj** [comment]
Hey,

sorry I didn't respond yet. No, sadly there was nothing in the logs that could help to solve this issue.

Could you please try connecting to this server?

https://paperless-demo.quinncasey.com/
paperless
paperless

**Comment by AdamGalt** [comment]
Done and sent! Let me know if you need any more and thank you for looking into this. 👍

**Comment by bauerj** [comment]
Hi Adam,

I still wasn't able to find the cause to this issue. Could you please provide me with some more information?

   - Do you connect to your Paperless instance through HTTPS?
   - Do you have a reverse proxy or anything in front of Paperless NG?
   - Can you run `curl -u 'username:password' https://paperless.example.com/api/documents/1/thumb/ -vvv --http1.0 --output 1.png; file 1.png` in a Linux console with your details filled in and send me the output? Do make sure to blank the line starting with "> Authorization" as that contains an encoded form of your credentials.

My suspicion is that there is something wrong with the server configuration as only two people reported about it. But it could also be a sneaky Flutter bug that only happens in a specific environment. Would you be able to test the app on a different Android device or in an emulator to see if that makes any difference?

Thank you for your help in debugging this and sorry again that this is taking so long.

**Comment by AdamGalt** [comment]
Hi Johann,

I'm not using https, just http in the house and direct tunneling through
tailscale outside the lan. Problem with the docs loading happens inside and
outside lan.
No reverse proxy or load balancer in front of the instance, but it is a
container.
I'm running the ghcr.io/linuxserver/paperless-ng 1.4.5 Docker image on an
Ubuntu 20.04.3 Server with Docker version 20.10.7

I've roped my wife into testing as well with her moto-G-9Play accessing it
without tailscale on the lan and she gets the same thing. Authentication is
successful but docs don't load.

Output:

*   Trying 100.109.54.79:8000...
* TCP_NODELAY set
  % Total    % Received % Xferd  Average Speed   Time    Time     Time
 Current
                                 Dload  Upload   Total   Spent    Left
 Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--
  0* Connected to 100.109.54.79 (100.109.54.79) port 8000 (#0)
* Server auth using Basic with user 'admin'
> GET /api/documents/1/thumb/ HTTP/1.0
> Host: 100.109.54.79:8000
> Authorization: Basic (REDACTED)
> User-Agent: curl/7.68.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
* HTTP 1.0, assume close after body
< HTTP/1.0 404 Not Found
< Content-Type: application/json
< Vary: Accept, Accept-Language, Origin
< Allow: GET, HEAD, OPTIONS
< X-Frame-Options: SAMEORIGIN
< X-Api-Version: 2
< X-Version: 1.4.5
< Content-Length: 23
< Content-Language: en-us
< X-Content-Type-Options: nosniff
< Referrer-Policy: same-origin
<
{ [23 bytes data]
100    23  100    23    0     0    139      0 --:--:-- --:--:-- --:--:--
140

------------------------------------------------------------------------------------------

Many Thanks,
Adam

**Comment by bauerj** [comment]
Can you confirm that a document with ID 1 exists by visiting http://100.109.54.79:8000/documents/1? If it doesn't, please repeat the command above with an ID of one of your documents instead of the `1`.

**Comment by AdamGalt** [comment]
Hi again Johann,

After doing that testing, I've gone to check the latest server version and
saw I'm a few point releases behind. I've pulled the latest stable
container 1.5.0 and everything is working as expected now!
Sorry for wasting your time, it was a bad container all along! Thank you
for all your patience and help.

Best wishes,
Adam

**Comment by bauerj** [comment]
Phew, thanks for letting me know 😄

Still a weird bug butty happy to hear you found a solution!

@Elderglass can you check if this solves the issue for you as well?

**Comment by thijselblaso** [comment]
Hi, for me this issue still exists! I have this problem only when I use the address of the proxy. Local I have no problems with the app. What do I expose over the proxy? I have  location /, /api/, /static/ and /ws/. Do I miss something?
